components/com_joomgallery/models/edit.php

```diff
@@ -502,6 +502,16 @@ public function delete()
     {
       JLog::add(JText::sprintf('COM_JOOMGALLERY_EDITIMAGE_MSG_COULD_NOT_DELETE_NAMETAGS', $this->_id), JLog::WARNING, 'jerror');
     }
+	
+	// Delete the corresponding database entries of the Votes
+    $query->clear();
+    $query->delete(_JOOM_TABLE_VOTES)
+          ->where('picid = '.$this->_id);
+    $this->_db->setQuery($query);
+    if(!$this->_db->query())
+    {
+      JLog::add(JText::sprintf('COM_JOOMGALLERY_EDITIMAGE_MSG_COULD_NOT_DELETE_VOTES', $this->_id), JLog::WARNING, 'jerror');
```

Comment:
Using ```JText::sprintf()``` is not neccessary here because your message does not contain any placeholders for variables, you could use ```JText::_()```` instead